java/CWE-434/UnrestrictedFileUpload.ql

```diff
@@ -0,0 +1,51 @@
+/**
+ * @name Unrestricted Upload of File with Dangerous Type
+ * @description Unrestricted Upload of File with Dangerous Type
+ * @id java/unrestricted-file-upload
+ * @kind path-problem
+ * @problem.severity error
+ * @security-severity 8.0
+ * @precision high
+ * @tags security
+ *       external/cwe/cwe-434
```

Comment:
java (`js/http-to-file-access`) and ruby (`rb/http-to-file-access`) tag these `cwe-912` as well